js/signup.js

```diff
@@ -0,0 +1,195 @@
+const querySelector = (selector) => document.querySelector(selector);
+
+const emailInput = querySelector("#email-input");
+const passwordInput = querySelector("#password-input");
+const emailErrorMessage = querySelector("#email-error-message");
+const passwordErrorMessage = querySelector("#password-error-message");
+const passwordConfirmInput = querySelector("#password-confirm-input");
+const passwordConfirmErrorMessage = querySelector("#password-confirm-error-message");
+const loginForm = querySelector("#login-form");
+
+const eyeOffButton = querySelector("#eye-off");
+const eyeOnButton = querySelector("#eye-on");
+const eyeOffConfirmButton = querySelector("#eye-off-confirm");
+const eyeOnConfirmButton = querySelector("#eye-on-confirm");
+
+const EMAIL_REGEX = /^[a-zA-Z0-9._-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,6}$/;
+const PASSWORD_REGEX = /^(?=.*[A-Za-z])(?=.*\d)[A-Za-z\d]{8,}$/;
+
+// 이메일 입력
+emailInput.addEventListener("focusout", validateEmailInput);
+
+function isValidateEmail(email) {
```

Comment:
요렇게 되면 조금 더 좋을 것 같네요!
```suggestion
function isValidEmail(email) {
```